ui/job-view/details/summary/ActionBar.jsx

```diff
@@ -477,6 +482,21 @@ class ActionBar extends React.PureComponent {
                   </a>
                 </li>
               ))}
+            <li>
+              <Button
+                id="find-job-btn"
+                title="Find this job instance"
```

Comment:
Preferred wording would be: "Scroll to selection"

Reply:
Fixed